reference/metadata_tables.rst

```diff
@@ -167,3 +173,65 @@ Citus manages shard health on a per-placement basis and automatically marks a pl
 |                |                      | | subsequent background activity.                                         |
 +----------------+----------------------+---------------------------------------------------------------------------+
 
+
+Worker Nodes table
+---------------------------------------
+
+The pg_dist_node table contains information about the worker nodes in the cluster. 
+
++----------------+----------------------+---------------------------------------------------------------------------+
+|      Name      |         Type         |       Description                                                         |
++================+======================+===========================================================================+
+| nodeid         |         int          | | Auto-generated identifier for an individual node.                       |
++----------------+----------------------+---------------------------------------------------------------------------+
+| groupid        |         int          | | Identifier used to denote a group of 1 primary and 0 or more            |
```

Comment:
Also _primaries_ and _secondaries_ are jargon-y and undefined. Maybe _leader_ and _followers_? Or to use PostgreSQL terminology, _primary server_ and _standby servers_?

At minimum, change to _primary server_ and _secondary servers_?